lib/publiccloud/azure.pm

```diff
@@ -475,7 +499,7 @@ sub terraform_apply {
     my ($self, %args) = @_;
     $args{vars} //= {};
     my $offer = get_var("PUBLIC_CLOUD_AZURE_OFFER");
-    my $sku = get_var("PUBLIC_CLOUD_AZURE_SKU");
+    my $sku = az_sku("");
```

Comment:
This `empty` default is here to avoid regression, reproducing same behavior as original code.